tools/requirements.txt

```diff
@@ -0,0 +1 @@
+-e git+https://github.com/dcos/shakedown.git@master#egg=shakedown
```

Comment:
This is really here to say that modify_master.py needs this.  I could just put it in the python that calls modify_master, or I could potentially put it in the modify_master module itself, though that would require it to delay imports (maybe fine).

Making the modules themselves able to declare what modules they need pip installed probably will require more magic than I'm super comfortable wedging into this pull request, but maybe theres a clear better state for right now.